tests/test_photon_counting.py

```diff
@@ -0,0 +1,145 @@
+import pytest
+import os
+import astropy.time as time
+import corgidrp.mocks as mocks
+import corgidrp
+from astropy.io import fits
+import numpy as np
+import corgidrp.walker as walker
+import corgidrp.caldb as caldb
+import corgidrp.data as data
+import corgidrp.detector as detector
+from corgidrp.photon_counting import get_pc_mean, photon_count, PhotonCountException
+
+detector_params = data.DetectorParams({})
+
+def test_expected_results():
```

Comment:
should this be an e2e test? It seems like it is testing the full use case like an e2e test. Another way to phrase the question: is there a different e2e test you were thinking of writing? 

A unit test would test just the inputs and outputs of the step function (rather than going through the whole walker infrastructure).

Reply:
I can do that!